python/lsst/summit/utils/nightReport.py

```diff
@@ -627,7 +627,7 @@ def plotPerObjectAirMass(
 
         objects = ensure_iterable(objects)
 
-        plt.figure(figsize=(10, 6))
+        plt.figure(figsize=(16, 12))
```

Comment:
What is the purpose of changing the figure size here and below? I wonder if there's a better fix, as this will probably have other consequences. I suspect these defaults were picked quite carefully (and they're also not the same, and you've changed them to be the same).

Reply:
it complains about figure size, I tried several bigger numbers and found those but maybe better numbers can be found (that feep the relation 10/6):
tests/test_nightReport.py::NightReportTestCase::test_plotPerObjectAirMass
  /sdf/home/c/cpio/development/summit_utils/python/lsst/summit/utils/nightReport.py:654: UserWarning: Tight layout not applied. The bottom and top margins cannot be made large enough to accommodate all axes decorations.
    plt.tight_layout()

Reply:
OK, thanks. It's not the most important plot to be honest, so let's leave it like this, and if it looks bad we can fix it up later, thanks.